src/_includes/header.html

```diff
@@ -53,6 +53,7 @@
 			  <li><a href="{{ site.baseurl }}/documentation/runners/flink/">Apache Flink Runner</a></li>
 			  <li><a href="{{ site.baseurl }}/documentation/runners/spark/">Apache Spark Runner</a></li>
 			  <li><a href="{{ site.baseurl }}/documentation/runners/dataflow/">Cloud Dataflow Runner</a></li>
+			  <li><a href="{{ site.baseurl }}/documentation/runners/apex/">Apache Apex Runner</a></li>
```

Comment:
Move Apex up, between Direct and Flink? (for fairness, alphabetical sort based on the full name, with the exception of direct runner)